client/app/cloud/project/billing/cloud-project-billing.less

```diff
@@ -7,6 +7,12 @@
   }
 }
 
+.cloud-billing-instance-list {
+  .change-billing-button-text {
```

Comment:
What's force you to set the font-size to 12px inside the column?

Is it related to the `<i class="fa fa-*">` element?

Reply:
The buttons appear inside one of the rows of datagrid. The text of second button is long and gets truncated so had to reduce the font size of it. The icon looks bigger if i dont reduce it too. So added the font-size to include both.

Reply:
Having multiple buttons inside a row of the datagrid is a requirement?

I mean that you could probably moved theses elements inside the [`<oui-action-menu>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/datagrid#actions) component.

In that way we avoid creating specific styles as much as we can.

Does it make sense?